### ISSUE TITLE
[Feature Request] GraphQl integration with Strawberry

### ISSUE THREAD

**Issue by sansyrox** [sansyrox/robyn#276]
**Current Behavior**

Right now, we have a basic version graphql support. Create a proper integration.

This integration is blocked view controller and views.

**Desired Behavior**

---
**Screenshots / Mockups**

**Alternatives**

---

**Comment by guilefoylegaurav** [comment]
Can I pick this up, @sansyrox? Seems interesting

**Comment by sansyrox** [comment]
@guilefoylegaurav , this issue is actually blocked on another issue right now. You can start with #296 if you find that interesting.

Also, if you have a windows machine set up. Would it be possible for you to help me out with the review of this PR(https://github.com/sansyrox/robyn/pull/294) ?

Are you on our discord btw?

**Comment by guilefoylegaurav** [comment]
Yes, sure, I can help you with the review of PR #294 
Also, yes, I am there on Robyn's discord

**Comment by sansyrox** [comment]
Thank you so much @guilefoylegaurav 😄 

I'll add you as a reviewer.

**Comment by sansyrox** [comment]
Thanks @XChikuX . This is WIP 😄

**Comment by sansyrox** [comment]
GraphQL integration with Strawberry is now documented end-to-end (graphql-support.mdx) with schema, GraphiQL, and query execution. Closing this; websocket-based subscriptions can be tracked in a fresh focused issue if still wanted.
